Example/SwiftSyft/HomeViewController.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  HomeViewController.swift
+//  SwiftSyft
+//
+//  Created by SashaBataieva on 01/20/2020.
+//  Copyright (c) 2020 mjjimenez. All rights reserved.
+//
+
+import UIKit
+
+enum StaticHomeScreenStrings {
+    static let headerDescription = "syft.js/grid.js testing"
+    static let description = """
+            This is a demo using @SwiftSyft@ from @OpenMined@ to execute a multi-worker protocol hosted on grid.js
+    """
+    static let swiftSyft = "SwiftSyft"
+    static let swiftSyftKey = "@SwiftSyft@"
+    static let swiftSyftURL = "https://github.com/OpenMined/SwiftSyft"
+    static let openMined = "OpenMined"
+    static let openMinedKey = "@OpenMined@"
+    static let openMinedURL = "https://github.com/OpenMined/grid.js"
+    static let socketURL = "ws://localhost:3000"
```

Comment:
Let's change "ws://localhost:300" to "wss://localhost:300" to convey that we want the user to use a secure protocol.

Reply:
We will use "wss://localhost:300" (I mean local host 300 not 3000) ?

Reply:
Oops, I meant 3000.